gradle-plugin/CHANGELOG.md

```diff
@@ -5,12 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
- 
+### Changed
+- Upgrade `com.android.tools.build:gradle` plugin to v3.3.2 (#41)
+
 ## [0.1] - 2019-01-15
 *Initial release.*
 
 ### Added
 - [ValidateAndroidAppReleaseConfiguration task](https://github.com/mozilla-mobile/android-automation-tools/blob/159757dbf2032ea374923e490806f12e0f765923/gradle-plugin/src/main/kotlin/org/mozilla/android/tasks/ValidateAndroidAppReleaseConfiguration.kt#L15-L23) to validate the repository is in an appropriate state before building a release build.
-  
+
```

Comment:
Nit: is this space intentional?

Reply:
I removed the extra space characters (as opposed to the intentional newlines) which is an intentional change.